### ISSUE TITLE
Some words are hidden in dark version 

### ISSUE THREAD

**Issue by adizh** [angular/angular#57448]
### Describe the problem that you experienced

In angular.dev, when switched to dark version some words are hidden, due to their color.

### Describe what you want to experience that would fix the problem

Just change the color of these words. As shown below, in light version these words are red, but in dark version they are not visible

### Add a screenshot if that helps illustrate the problem

<img width="946" alt="Screen Shot 2024-08-19 at 16 00 18" src="https://github.com/user-attachments/assets/28883a33-4065-4cb2-bb15-656514894642">
<img width="929" alt="Screen Shot 2024-08-19 at 16 00 39" src="https://github.com/user-attachments/assets/77c95caa-f623-4abd-9a40-8eb38e0b8ba2">

**Comment by JeanMeche** [comment]
It looks like you're using a rather old os/browser. Could you confirm ?

**Comment by adizh** [comment]
> It looks like you're using a rather old os/browser. Could you confirm ?

Safari Version 15.6

**Comment by codevby** [comment]
Is the problem occurring in other browsers as well?

**Comment by JeanMeche** [comment]
@codevby Yes, you'll see this issue on browser that don't suppor `color-mix` : https://caniuse.com/?search=color-mix

**Comment by adizh** [comment]
> Is the problem occurring in other browsers as well?

in my chrome it is ok, only in safari

**Comment by JeanMeche** [comment]
This is a browser target specific issue. Haven't had more report since.
